src/OrchardCore/OrchardCore/Modules/ModularTenantRouterMiddleware.cs

```diff
@@ -97,6 +97,8 @@ private IShellPipeline BuildTenantPipeline()
                 ConfigureTenantPipeline(builder);
             };
 
+            var test = startupFilters.Reverse();
```

Comment:
Where the `test` is used?!!

Reply:
See my commits, i removed this testing code just after ;)